A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js

```diff
@@ -325,7 +350,7 @@ class A320_Neo_PFD_Attitude extends NavSystemElement {
             }
 
             this.hsi.setAttribute("slip_skid", Simplane.getInclinometer().toString());
-            this.hsi.setAttribute("radio_altitude", Simplane.getAltitudeAboveGround(true).toString());
+            this.hsi.setAttribute("radio_altitude", Simplane.getAltitudeAboveGround().toString());
```

Comment:
was this 'true' removed here intentionally?

Reply:
No, adding it back